adapters/handlers/grpc/v1/filters.go

```diff
@@ -257,10 +260,10 @@ func extractPath(className string, on []string) (*filters.Path, error) {
 	return &filters.Path{Class: schema.ClassName(className), Property: schema.PropertyName(on[0]), Child: nil}, nil
 }
 
-func extractPathNew(getClass func(string) *models.Class, className string, target *pb.FilterTarget, operator filters.Operator) (*filters.Path, schema.DataType, error) {
-	class := getClass(className)
-	if class == nil {
-		return nil, "", fmt.Errorf("could not find class %s in schema", className)
+func extractPathNew(getClass func(string) (*models.Class, error), className string, target *pb.FilterTarget, operator filters.Operator) (*filters.Path, schema.DataType, error) {
+	class, err := getClass(className)
+	if err != nil {
+		return nil, schema.DataTypeString, err
```

Comment:
Same here, why the change from `return nil, "", err` to `return nil, schema.DataTypeString, err`?